merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/MissionModelService.java

```diff
@@ -50,6 +50,11 @@ Map<String, SerializedValue> getActivityEffectiveArguments(String missionModelId
          NoSuchActivityTypeException,
          InstantiationException;
 
+  List<BulkEffectiveArgumentResponse> getActivityEffectiveArgumentsBulk(
```

Comment:
Have you considered replacing `getActivityEffectiveArguments` with `getActivityEffectiveArgumentsBulk` at the `MissionModelService` layer?

Reply:
Additionally, maybe the existing `getActivityEffectiveArguments` endpoint could be marked as
```gql
@deprecated(reason: "Use bulk query getActivityEffectiveArgumentsBulk instead")
```

Reply:
The latest commit I've added deprecates the existing `getActivityEffectiveArguments` Hasura action in favor of the new bulk query action. The existing endpoint still exists, but will now return a list of `EffectiveArgumentResponse`, which is a small breaking change.

Additionally, this fixes the above bug where invalid activity type names were throwing an unserialized server error, as `getActivityEffectiveArguments` now uses the new serialization introduced with the new `getActivityEffectiveArgumentsBulk` action. I could make this a non-breaking change by adding some extra glue code, if that is desired.

Reply:
Hmm, I was imagining we could get away without a breaking change 🤔 In theory there can be two separate endpoints in `MerlinBindings` that call the same method in `MerlinService`. Ah, I guess that's what you mean by "adding some extra glue code".

If possible, I'd like to make this a smooth deprecation - give users a release or two to use the new endpoint in their scripts, and then pull the rug

Reply:
Good call. Turns out the only "glue code" I needed was calling a function I already wrote that serializes a single response 🧠. Change is no longer breaking, and we can remove the deprecated endpoint (possibly along with `getSequenceSeqJson` and `getEdslForSeqJson`) in a later release.